crates/holochain_sqlite/src/db.rs

```diff
@@ -380,6 +381,7 @@ impl<Kind: DbKindT + Send + Sync + 'static> DbWrite<Kind> {
             .expect("Database transaction failed")
     }
 
+    // TODO never used and has a warning that you need to be careful with it, remove?
```

Comment:
I vote to go ahead and remove. (heh, it won't let me make a suggestion to do so, because the body of the fn isn't in the diff : /)

Reply:
Same response as above, I agree and will remove it on the very next PR